### ISSUE TITLE
[Terminal >Actions]: Screen Reader users are not able to know the purpose of Actions List inside Actions tab page.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12057]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win + Ctrl + Enter' keys.
4. Activate 'Actions' tab and then navigate to actions list using tab/arrow keys and observe the issue.

**User Experience:**
Screen Reader users UX will not be good as they are not able to know the purpose of list item when they navigate to them.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Screen Reader users are not able to know the purpose of Actions List inside Actions tab page..zip](https://github.com/microsoft/terminal/files/7783331/Screen.Reader.users.are.not.able.to.know.the.purpose.of.Actions.List.inside.Actions.tab.page.zip)

### Expected Behavior

Screen Reader users should be able to know the purpose of Actions List inside Actions tab page.
i.e. When user first time jumps to 'Actions' list item control, Screen Reader should announce something like this 'Actions List item, Use these actions to perform available functionality inside Terminal Text Area....'

### Actual Behavior

Screen Reader users are not able to know the purpose of Actions List inside Actions tab page.

**Observation:** Same issue repro with NVDA and JAWS Screen Reader.

**Comment by carlos-zamora** [comment]
> "Actions List item, Use these actions to perform available functionality inside Terminal Text Area...."

Having trouble figuring out how to do this. Here's a few notes:
- add an `AutomationProperties.Name` to the `ListView`:
   -  Name is read when focus moves inside the list. So if it's set to "Actions", moving to the first item says "Actions <name of action> <key chord>".
   - Issue: it's weird that Narrator doesn't say the control type "list". NVDA does so I'll just assume this part is ok. Otherwise, if I name it "Actions list", NVDA will say "Actions list list"
- how do you get it to say "Use these actions..."?
   - Help text on the `ListView` isn't automatically read.
   - Help text on the action seems like the wrong place.
   - It should be a property on the `ListView`, but I don't know which one.

I (or someone) will circle back around to fix this bug.
